.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - uses: avsm/setup-ocaml@master
+        with:
+          ocaml-version: ${{ matrix.ocaml }}
```

Comment:
You can probably delete this whole `with:` clause. There is no matrix of OCaml versions for this job. `setup-ocaml` will pick some recent version automatically.